precompiles/staking/IStaking.json

```diff
@@ -0,0 +1,155 @@
+{
+  "abi": [
+    {
+      "inputs": [],
+      "name": "getAllValidators",
+      "outputs": [
+        {
+          "components": [
+            {
+              "internalType": "string",
+              "name": "operatorAddress",
+              "type": "string"
+            },
+            {
+              "internalType": "string",
+              "name": "consensusPubKey",
+              "type": "string"
+            },
+            {
+              "internalType": "bool",
+              "name": "jailed",
+              "type": "bool"
+            },
+            {
+              "internalType": "enum BondStatus",
+              "name": "bondStatus",
+              "type": "uint8"
+            }
+          ],
+          "internalType": "struct Validator[]",
```

Comment:
I think we might encounter issues if validators array is too large.

 UniswapV2  factory solves this by having two methods:
- `allPairsLength: int`
- `allPairs(num: int): Pair`

Should we consider something similar? @lumtis Number of validators can go to several hundreds / thousands

Reply:
Would not go more than 150